test/sqllogictest/role.slt

```diff
@@ -243,3 +243,47 @@ SELECT current_user();
 ----
 db error: ERROR: role u4 was concurrently dropped
 DETAIL: Please disconnect and re-connect with a valid role.
+
+simple conn=mz_system,user=mz_system
+DROP CLUSTER mz_system CASCADE;
+----
+db error: ERROR: system cluster 'mz_system' cannot be modified
+
+simple conn=mz_system,user=mz_system
+DROP CLUSTER mz_introspection CASCADE;
+----
+db error: ERROR: system cluster 'mz_introspection' cannot be modified
+
+# TODO: I'm expecting this to fail, why does it succeed in SLT?
+# db error: ERROR: system cluster 'mz_system' cannot be modified
```

Comment:
Let's open a bug for this, it looks like a mistake.

Reply:
The other TODOs here as well.

Reply:
Ahh ok, actually I'm wrong. Only the `mz_system` and `mz_introspection` roles can delete and create cluster replicas in system clusters. Since you're using `mz_system`, this is expected to work.